utils/getName.test.js

```diff
@@ -61,4 +61,15 @@ describe("getName", () => {
       expect(getName(entity, typeName)).toEqual(entity.label)
     );
   });
+
+  it("should use key if none of the other fields are avaialable", () => {
```

Comment:
Should this not be part of the `should use correct key` test?  Also, the wording for this scenario will become redundant if we add another fallback key.

Reply:
`should use correct key` doesn't test the precedence - just the field to use first.